dfsclient/src/main/java/org/hpccsystems/dfs/client/RowServiceInputStream.java

```diff
@@ -1583,50 +1508,11 @@ public void close() throws IOException
     @Override
     public void mark(int readLim)
     {
-        if (CompileTimeConstants.PROFILE_CODE)
```

Comment:
are we dropping all profiling code?

Reply:
No, the other profiling code will remain, I removed code that tracking if the buffer mutex was being held for too long because it was no longer useful.